parser/src/python.lalrpop

```diff
@@ -150,7 +152,8 @@ ExpressionStatement: ast::Stmt = {
 };
 
 AssignSuffix: ast::Expr = {
-    "=" <e:TestListOrYieldExpr> => e
+    "=" <e:TestListOrYieldExpr> => e,
+    "=" <e:LineMagicExpr> => e
 };
 
 TestListOrYieldExpr: ast::Expr = {
```

Comment:
We could merge this into `TestListOrYieldExpr` but then the name would have to change and will divert with the convention:

```lalrpop
TestListYieldOrLineMagicExpr: ast::Expr = {
	TestList,
	YieldExpr,
	LineMagicExpr
};
```